elasticdl_preprocessing/tests/to_ragged_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import tensorflow as tf
+
+from elasticdl_preprocessing.layers.to_ragged import ToRagged
+from elasticdl_preprocessing.tests.test_utils import ragged_tensor_equal
+
+
+class ToRaggedTest(unittest.TestCase):
+    def test_dense_to_ragged(self):
+        layer = ToRagged()
+        inp = tf.constant([[1], [-1], [4]], tf.int64)
+        out = layer(inp)
+        expected_out = tf.ragged.constant([[1], [], [4]], tf.int64)
+        self.assertTrue(ragged_tensor_equal(out, expected_out))
```

Comment:
I find that `RaggedTensor` has a member function `to_list`. You can convert the tensor to a numpy list and use a normal `assertListEqual` to compare them. I think those function will give more information when anything is wrong. BTW, I have not check the code for `ragged_tensor_equal`.

Reply:
`ragged_tensor_equal` checks the dtypes and contents of `RaggedTensor`. It is more simple to check the contents by `to_list` than the implemention in `ragged_tensor_equal`. I will use `to_list` to compare. Thank you for the suggestion!